ccm_web/server/src/api/api.controller.ts

```diff
@@ -15,7 +17,7 @@ export class APIController {
   }
 
   @Get('globals')
-  getGlobals (): Globals {
-    return this.apiService.getGlobals()
+  getGlobals (@Req() req: Request, @Session() session: SessionData): Globals {
```

Comment:
Do we need `Req` or `Request` yet in this file, if you're just going to use `session`?

Reply:
Removed the Request.